playground/frontend/lib/pages/playground/states/examples_state.dart

```diff
@@ -82,31 +87,41 @@ class ExampleState with ChangeNotifier {
     return example;
   }
 
-  _loadCategories() async {
-    sdkCategories = await _exampleRepository.getListOfExamples(
-      GetListOfExamplesRequestWrapper(sdk: null, category: null),
-    );
-    await _loadDefaultExamples(sdkCategories);
-    notifyListeners();
+  _loadCategories() {
+    _exampleRepository
+        .getListOfExamples(
+          GetListOfExamplesRequestWrapper(sdk: null, category: null),
+        )
+        .then((map) => setSdkCategories(map));
   }
 
   changeSelectorVisibility() {
     isSelectorOpened = !isSelectorOpened;
     notifyListeners();
   }
 
-  _loadDefaultExamples(sdkCategories) async {
+  loadDefaultExamples() async {
+    if (defaultExamplesMap != null) {
+      return;
+    }
```

Comment:
Nit: add a new line after the return block.

Reply:
Sure, I'll add a new line in the next commit.